nubis/puppet/files/rc.local

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
Good catch, I removed that bit on purpose, because it was redundant. confd's own startup takes place after consul has started (in /etc/nubis.d/confd) and does this already.